plugins/workspace-minimap/src/minimap.ts

```diff
@@ -27,44 +27,55 @@ const BlockEvents = new Set([
  * your code looks like, and how it is organized.
  */
 export class Minimap {
-    protected primaryWorkspace: Blockly.WorkspaceSvg;
-    protected minimapWorkspace: Blockly.WorkspaceSvg;
+    private primaryWorkspace: Blockly.WorkspaceSvg;
+    minimapWorkspace: Blockly.WorkspaceSvg;
```

Comment:
Do you want the minimap workspace to be public?

It looks like the only things you're doing are setting the CSS of the injection div and triggering a resize! You can probably create methods for those on the minimap class and keep the `minimapWorkspace` `private` or `protected.

Reply:
Great idea will do that